notebooks/reinforcement_learning/solutions/contextual_bandits_with_tf_agents.ipynb

```diff
@@ -0,0 +1,1813 @@
+{
```

Comment:
<p>note really anything we can do but the Error statements in the output when the code is executing is quite confusing. Made me think there was something was wrong in the code.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/160/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
<p>I agree.</p>